.github/workflows/selenium-lab-tests.yaml

```diff
@@ -0,0 +1,93 @@
+name: Selenium Lab Tests
+
+on:
+  workflow_dispatch:
+    # Allows for manual triggering on PRs.  They should be reviewed first, to
```

Comment:
Not that I'm aware of.

It's a short list of people who can trigger runs in the lab (only Googlers at this moment, but other maintainers soon), and the only purely automated triggers are for code that has already been reviewed and merged.

The lab is at Google, but not a part of the Google corporate network.  The self-hosted runner used here is ephemeral and containerized, and will be wiped and recreated between test runs.

We're still protected against malicious PRs (since a maintainer has to review and trigger), but not against malicious maintainers.  But we were never protected against malicious maintainers, so this isn't obviously any worse.

Reply:
Oh, maybe this is a dumb question, but would it be possible for a non-maintainer to call this from their fork? That could make the problem of malicious maintainers more serious.

Reply:
Yes, but not in our lab.  It runs on a self-hosted runner, which is registered to the org.  A repo outside of shaka-project can't access our runner.  A fork could set up their own self-hosted runner and use this as much as they want.  In fact, I plan to document the setup, so that our lab environment could be replicated by others if they want.